app/controllers/fe/applications_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Fe
   class ApplicationsController < ApplicationController
```

Comment:
I guess the thought there was sp/si/pr all have to do something similar for this functionality.. but I think it would make sense just to let those apps define their own ApplicationController functionality.  They already customize a lot of it.